modules/dr-restoring-cluster-state.adoc

```diff
@@ -304,3 +304,21 @@ etcd-ip-10-0-173-171.ec2.internal                2/2     Running     0
 ----
 
 Note that it might take several minutes after completing this procedure for all services to be restored. For example, authentication by using `oc login` might not immediately work until the OAuth server pods are restarted.
+
+[id="dr-scenario-2-restoring-cluster-state_storage_{context}"]
+= Issues and workarounds in restoring persistent storage state
+
+If your {product-title} uses persistent storage of any form, typically a state of the world is stored outside etcd. It could be a ElasticSearch
+cluster running in a Pod or a database running in a statefulset. When {product-title} is restored from etcd backup, we are also restoring status of those workloads in Openshift. But depending on how old the etcd snapshot is - that status may be invalid or outdated.
+
+Some examples are:
+
+1. Mysql database was running in a Pod backed up by a PV object. But after the etcd snapshot was taken - the Mysql Pod and corresponding PV were deleted and hence underlying volume on storage provider would be deleted. Restoring Openshift from etcd snapshot is not going to bring back the volume on storage provider and hence is not going to result in running Mysql Pod(even though Pod will try to start but will keep failing) and a manual action must be taken to restore mysql Pod. In this case manual action could be restoring the volume on storage provider via volume snapshots and then editing PV object to point to new volume.
+
+2. You have a pod P1 running using volume A which is attached to node X. If the etcd snapshot is taken and another pod uses the same volume on node Y and a etcd restore is performed then pod P1 may not able to start correctly because volume is still attached to node Y but {product-title} does know about it and hence is not going to detach it. In this case volume must be manually detached from node Y, so as volume can attach on node X and pod P1 can start.
```

Comment:
The second and third sentences are a bit clunky. I'd suggest rewording it to be - "If the etcd snapshot is taken while another pod uses the same volume on node Y, then when the etcd restore is performed pod P1 may not be able to start correctly due to the volume still being attached to node Y. {product-title} is aware of the attachment, and will not automatically detach it." 

Last sentence -"...case the volume".

Reply:
Working with Christian's rewrite:
"Pod P1 is running using volume A, which is attached to node X. If the etcd snapshot is taken while another pod is using the same volume on node Y, then when the etcd restore is performed pod P1, it may not be able to start correctly due to the volume still being attached to node Y. {product-title} is aware of the attachment, and does not automatically detach it. When this occurs, the volume must be manually detached from node Y, so the volume can attach on node X, and then pod P1 can start."